src/templates/it/base.html

```diff
@@ -0,0 +1,170 @@
+{% block index_title %}Indice{% endblock %}
+{% block open_the_index %}Apri sommario{% endblock %}
+{% block close_the_index %}Chiudi sommario{% endblock %}
+
+{% block and %}e{% endblock %}
+{% block comma %}, {% endblock %}
+
+{% block open %}Apri{% endblock %}
+{% block close %}Chiudi{% endblock %}
+
+{% macro figure_text(metadata, id) %}Figura {{ figure_id(metadata, id) }}.{% endmacro %}
+{% macro show_description(metadata, id) %}Mostra la descrizione della Figura {{ figure_id(metadata, id) }}{% endmacro %}
+{% macro hide_description(metadata, id) %}Nascondi la descrizione della Figura {{ figure_id(metadata, id) }}{% endmacro %}
+
+{% block written_by_before %}Scritta da{% endblock %}
+{% block reviewed_by_before %}Revisionata da{% endblock %}
+{% block translated_by_before %}Tradotta da{% endblock %}
+{% block analysis_by_before %}Analizzata da{% endblock %}
+{% block written_by_after %}{% endblock %}
+{% block reviewed_by_after %}{% endblock %}
+{% block translated_by_after %}{% endblock %}
+{% block analysis_by_after %}{% endblock %}
+
+{% block unedited %}[Inedito]{% endblock %}
+
+{% block author %}Autore{% endblock %}
+{% block authors %}Autori{% endblock %}
+
+{% macro onTwitter(twitterHandle) %}@{{twitterHandle}} su Twitter{% endmacro %}
+{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} su GitHub{% endmacro %}
+{% macro onLinkedIn(authorName) %}{{authorName}} su LinkedIn{% endmacro %}
+{% macro website(authorName) %}{{authorName}} sito web{% endmacro %}
+
+{% macro edition(year) %}Edizione {{ year }}{% endmacro %}
+
+{% block footer_title %}Navigazione nel footer di pagina{% endblock %}
+{% block copyright %}© Web Almanac. Concesso in licenza da <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/LICENSE">Apache 2.0</a>.{% endblock %}
+{% block accessibility_statement %}Dichiarazione di accessibilità{% endblock %}
+
+{% block featured_chapter %}Capitolo in primo piano{% endblock %}
+{% block featured_chapter_last_year %}Capitolo in primo piano<br/>dal Web Almanac {{ year|int -1 }}{% endblock %}
+
+{# Check if read_chapter already defined in child template as macros can't be overridden #}
+{% if not read_chapter %}
+{% macro read_chapter(chapter) %}Leggi il capitolo: <span class="featured-chapter-name">{{ chapter }}</span>{% endmacro %}
+{% endif %}
+{% if not read_last_years_chapter %}
+{% macro read_last_years_chapter(chapter) %}Leggi il capitolo {{ year|int-1 }} <span class="featured-chapter-name">{{ chapter }}</span>{% endmacro %}
+{% endif %}
+
+{% block contributors_description %}
+The Web Almanac has been made possible by the hard work of the web community. {{ self.contributors() }} people have volunteered countless hours in the planning, research, writing and production phases.
+Il Web Almanac è stato reso possibile grazie al duro lavoro della web community. {{self.contributors() }} persone si sono offerte volontarie per innumerevoli ore nelle fasi di pianificazione, ricerca, scrittura e produzione.
```

Comment:
Delete the English part:

```suggestion
Il Web Almanac è stato reso possibile grazie al duro lavoro della web community. {{self.contributors() }} persone si sono offerte volontarie per innumerevoli ore nelle fasi di pianificazione, ricerca, scrittura e produzione.
```